cylc/sphinx_ext/cylc_lang/autodocumenters.py

```diff
@@ -106,7 +106,7 @@ def directive(
     return ret
 
 
-def repr_value(value):
+def repr_value(value, vdr=None):
```

Comment:
I think either a clearer arg name or docstring entry would help

Reply:
vdr is the standard abbreviation, added a docstring.